### PR TITLE
All claims 1dot3 mg

### PR DESCRIPTION
**Pull request by markgreenburg**
## Description
Adds 1.3 to the all claims 526 form. This includes two pages:
- whether the veteran receives military retired pay, and if so, what branch gives it to him or her
- If veteran receives military pay, a second page asks the veteran whether or not to waive VA pay

## Testing done
- Tested locally
- Added some unit tests to cover requirements 

## Testing Plan
- Test in staging manually
- Test in production when possible

## Screenshots
First page base questions:
screen shot 2018-08-21 at 2 35 43 pm

-----

First page expands if the veteran answers 'yes':
screen shot 2018-08-21 at 2 35 57 pm
-----
screen shot 2018-08-21 at 2 36 09 pm

-----

Second page shows only if veteran selected 'yes' on first page:
screen shot 2018-08-21 at 2 36 27 pm

## Acceptance Criteria (Definition of Done)
- Veteran is able to answer all three questions (receive retired pay, what branch, and waive VA pay)
- Require all three questions when first answer is 'yes'
- Skip next two questions if first answer is 'no'

#### Applies to all PRs

- [x] Appropriate logging
- [x] Documentation has been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/pages/alternateNames.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const uiSchema = {
     'ui:widget': 'yesNo'
   },
   alternateNames: {
-    'ui:description': 'What names did you serve under?',
+    'ui:description': 'What name did you serve under?',
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/pages/alternateNames.js]: This should've been part of the 1.2 PR - this change reflects the latest copy. Just FYI.

File: src/applications/disability-benefits/all-claims/config/schema.js [hunk 2]
```diff
@@ -34,6 +34,21 @@ const schema = {
...
       }
+    },
+    militaryRetiredPayBranch: {
+      type: 'string',
+      'enum': [
+        'Air Force',
+        'Army',
+        'Coast Guard',
+        'Marine Corps',
+        'National Oceanic and Atmospheric Administration',
```
Comment (jbalboni) [inline src/applications/disability-benefits/all-claims/config/schema.js]: This is abbreviated in some of the other branch lists, does EVSS validate these branch values?

**Review by jbalboni — APPROVED** [review]
LGTM, had one question.

File: src/applications/disability-benefits/all-claims/config/schema.js (hunk 2 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/all-claims/config/schema.js]: Yes, they do, and they require a very strange format. Luckily, vets-api is doing this translation for us. In the Increase form, this is abbreviated, but I decided to just use the provided copy for this issue. I suppose one of these days we should probably decide whether we want to abbreviate it or spell it out across the board.

**Comment by markgreenburg** [comment]
Going to just merge with Jeff's approval to unblock work on 1.4, which would have some messy merge conflicts without 1.3 already in. We'll be sure to circle back on a final content review for chapter 1 once the chapter tickets are done.

**Comment by peggygannon** [comment]
@markgreenburg Hi, Mark.. it looks like this PR already got merged in. do you want me to give you my edits for this screen here or in another PR?

**Pull request merged**